WATOLINK/bci-gaming
# Move Player Health Functionality into Player class

## ClickUp ticket link

[Move playerHealth functionality into Player class](https://app.clickup.com/t/8686fhtyj)



## Implementation description

Player health and heal script related code was migrated into the Player class. 


## Steps to test
1. Make sure the health bar image reduces health when player is hit
2. Make sure player is able to be controled
3. Make sure the "enemy hit" log works



## What should reviewers focus on?

The steps to test and check on the heal script primarily. 

## Checklist
